packages/salesforcedx-vscode-core/src/commands/forceApexClassCreate.ts

```diff
@@ -48,8 +48,8 @@ class SelectFileName implements ParametersGatherer<{ fileName: string }> {
 class SelectDirPath implements ParametersGatherer<{ outputdir: string }> {
   private explorerDir: string | undefined;
 
-  public constructor(explorerDir?: { path: string }) {
-    this.explorerDir = explorerDir ? explorerDir.path : explorerDir;
+  public constructor(explorerDir?: { fsPath: string }) {
```

Comment:
What is the actual type of explorerDir? Are you able to use the `typeof` keyword as is done in https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/predicates/salesforcePredicates.ts#L18 to get a more precise type rather than doing a partial one like `{fsPath: string}`?